x-pack/plugins/cloud_security_posture/public/components/csp_page_template.tsx

```diff
@@ -94,12 +100,20 @@ const getPackageNotInstalledNoDataConfig = (
   },
 });
 
-const DefaultLoading = () => <CspLoadingState>{LOADING}</CspLoadingState>;
+const DefaultLoading = () => (
+  <CspLoadingState data-test-subj={LOADING_STATE_TEST_SUBJECT}>
+    <FormattedMessage
+      id="xpack.csp.cspPageTemplate.loadingDescription"
+      defaultMessage="Loading..."
```

Comment:
_mega microcopy nit_ - I would remove the `...` i think it's a bit old fashioned